doc/api/crypto.md

```diff
@@ -1539,6 +1539,73 @@ This should normally never take longer than a few milliseconds. The only time
 when generating the random bytes may conceivably block for a longer period of
 time is right after boot, when the whole system is still low on entropy.
 
+### crypto.randomFillSync(buf[, offset][, size])
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `buf` {Buffer}
+* `offset` {Number}
+* `size` {Number}
+
+This function is similar to [`crypto.randomBytes()`][] but requires the first
+argument to be a [`Buffer`][] that will be filled instead of returned. Another
```

Comment:
I definitely agree with @sam-github's suggestion that these ought to return the instance of the `Buffer` passed in so that calls can be chained.